src/main/groovy/net/vivin/gradle/versioning/tasks/TagTask.groovy

```diff
@@ -13,11 +14,24 @@ import org.gradle.tooling.BuildException
  * @author vivin
  */
 class TagTask extends DefaultTask {
+
+    Closure<String> tagMessage = {
+        "v${project.version}"
```

Comment:
When we spoke about the default value we said an annotated tag without message will be our default, as this message gives no added value at all, just repeats the info that is already present in the tag name. Also it is more consistent to what we have until now and it is easily configurable if someone wants it like that with `tag { message { "v$version" } }`.